components/alert/alert.js

```diff
@@ -66,19 +68,33 @@ class Alert extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 
 			.d2l-alert-text {
 				flex: 1;
-				padding: 1rem 1.2rem 1rem 1.5rem;
+				padding: 0.9rem 1.5rem;
 				position: relative;
 			}
-			:host([dir="rtl"]) .d2l-alert-text {
-				padding-left: 1.2rem;
-				padding-right: 1.5rem;
+			.d2l-alert-text-with-actions {
+				padding-right: 0.9rem;
+			}
+
+			:host([dir="rtl"]) .d2l-alert-text-with-actions {
+				padding-left: 0.9rem;
 			}
+
 			.d2l-alert-subtext {
 				margin: 0.5rem 0 0;
 			}
 
 			.d2l-alert-action {
-				margin: 0.3rem;
+				margin: 0.6rem;
```

Comment:
I think the design calls for only the 18px between the actions and the text, so with this going all the way around it ends up with (0.9rem + 0.6rem) = 1.5rem = 30px instead of 18.

Reply:
We probably could! 🤔